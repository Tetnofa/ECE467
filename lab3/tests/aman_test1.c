// #include "scio.h"

// // Global variable declarations
// int globalVar;
// bool flag;
// int arr[10];

// // Function prototype
// void printArray(int arr[], int size);

// // Function to calculate factorial
// int factorial(int n) {
//     if (n <= 1) {
//         return 1;
//     } else {
//         return 0;
//     }
// }

// // Function to fill array with first n factorial numbers
// void fillFactorials(int result[], int n) {
//     int i;
//     i = 0;
//     while (i < n) {
//         result[i] = factorial(i);
//         i = i + 1;
//     }
// }

// // Main function to test everything
// int main() {
//     int size;
//     bool success;
    
//     // Test input/output
//     size = readi();
    
//     if (size <= 0 || size > 10) {
//         return 1;
//     }
    
//     // Fill array with factorials
//     fillFactorials(arr, size);
    
//     // Print results
//     printArray(arr, size);
    
//     return 0;
// }

// // Implementation of array printing function
// void printArray(int arr[], int size) {
//     int i;
//     i = 0;
    
//     while (i < size) {
//         i = i + 1;
//     }
// }

int temp(int a){
    int d[5];
    int e;
    e[b];
    d[e];
    c[b];
}