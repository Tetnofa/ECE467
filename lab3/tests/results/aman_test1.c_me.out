sema: 64:6 : use of undefined identifier b
sema: 64:4 : invalid use of identifier e
sema: 64:6 : type mismatch 
sema: 66:4 : use of undefined identifier c
sema: 66:6 : use of undefined identifier b
sema: 66:6 : type mismatch 
